fuse_doc/doc/getting_started.rst

```diff
@@ -0,0 +1,253 @@
+    world_frame_id: "odom"
+    publish_tf: true
+    publish_frequency: 10
+
+Here, we configure the plugin that will publish our state estimate. The `fuse_publishers::Odometry2DPublisher` publishes a ROS `nav_msgs/Odometry` message, as well as a transform from the frame specified in the `world_frame` parameter to the frame specified in the `base_link_frame_id` parameter.
+
+- The `topic` is the ROS topic on which the output will be published.
+- The `*_frame_id` parameters specify the various coordinate frame IDs that will be used when publishing the `nav_msgs/Odometry` message.
+- The `publish_tf` parameter can be used to enable or disable publishing the transform for use by `tf2`.
+
+Adding a Second Sensor
+**********************
+
+The example so far fuses only a single odometry source, which isn't especially useful. In order to benefit from actual sensor fusion, we should add another sensor. In this case, we will add an IMU. We will augment our existing configuration.
+
+.. code-block:: yaml
+
+  # Fixed-lag smoother configuration
+  optimization_frequency: 20
+  transaction_timeout: 0.01
+  lag_duration: 0.5
+
+  solver_options:
+    max_num_iterations: 5
+
+  motion_models:
+    unicycle_motion_model:
+      type: fuse_models::Unicycle2D
+
+  unicycle_motion_model:
+    #                         x      y      yaw    vx     vy     vyaw   ax   ay
+    process_noise_diagonal: [0.100, 0.100, 0.100, 0.100, 0.100, 0.100, 0.1, 0.1]
+    scale_process_noise: true
+
+  sensor_models:
+    initial_localization_sensor:
+      type: fuse_models::Unicycle2DIgnition
+      motion_models: [unicycle_motion_model]
+      ignition: true
+    odometry_sensor:
+      type: fuse_models::Odometry2D
+      motion_models: [unicycle_motion_model]
+    imu_sensor:
+      type: fuse_models::Imu2D
+      motion_models: [unicycle_motion_model]
+
+  initial_localization_sensor:
+    publish_on_startup: true
+    #                x      y      yaw    vx     vy     vyaw    ax     ay
+    initial_state: [0.000, 0.000, 0.000, 0.000, 0.000, 0.000, 0.000, 0.000]
+    initial_sigma: [0.100, 0.100, 0.100, 0.100, 0.100, 0.100, 0.100, 0.100]
+
+  odometry_sensor:
+    topic: "odom"
+    twist_target_frame: "base_footprint"
+    linear_velocity_dimensions: ['x', 'y']
+    angular_velocity_dimensions: ['yaw']
+
+  imu_sensor:
+    topic: "imu"
+    angular_velocity_dimensions: ['yaw']
+    linear_acceleration_dimensions: ['x', 'y']
+    twist_target_frame: "base_footprint"
+
+  publishers:
+    filtered_publisher:
+      type: fuse_models::Odometry2DPublisher
+
+  filtered_publisher:
+    topic: "odom_filtered"
+    base_link_frame_id: "base_footprint"
+    odom_frame_id: "odom"
+    map_frame_id: "map"
+    world_frame_id: "odom"
+    publish_tf: true
+    publish_frequency: 10
+
+Note that we have added an `imu_sensor` section to `sensor_models`, and then specified the parameters for that new model.
+
+- The `topic` specifies the topic on which to listen for the IMU data.
```

Comment:
```suggestion
- The `topic` specifies the topic on which to listen for the `sensor_msgs/IMU` data.
```

Or something similar to mention the actual topic type supported.

Reply:
Yes, will add. Thanks.